webapp/locales/de.json

```diff
@@ -515,7 +515,11 @@
     "search": {
         "placeholder": "Suchen",
         "hint": "Wonach suchst Du?",
-        "failed": "Nichts gefunden"
+        "failed": "Nichts gefunden",
+        "heading": {
+          "Post": "Beiträge",
+          "User": "Benutzer"
+        }
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-07T11:29:38Z" title="Tuesday, January 7th 2020, 12:29:38 pm +01:00">Jan 7, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -515,7 +515,12 @@
     "search": {
         "placeholder": "Suchen",
         "hint": "Wonach suchst Du?",
-        "failed": "Nichts gefunden"
+        "failed": "Nichts gefunden",
+        "heading": {
+          "Post": "Beiträge",
+          "User": "Benutzer",
+	  "Tag": "Hashtags"
```


```suggestion
	    "Tag": "Hashtags"
```

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-07T11:30:07Z" title="Tuesday, January 7th 2020, 12:30:07 pm +01:00">Jan 7, 2020</time>_

---


the indentation is incorrect here, and we don't need this in this `PR`, aka code creep